my-app/src/store/models/CoinMarketData.ts

```diff
@@ -0,0 +1,42 @@
+import {
+  CurrenciesApi,
+  CurrenciesModels,
+  normalizeCurrencies,
+} from "@store/models/Currencies";
+
+export type CoinMarketDataApi = {
+  market_cap: CurrenciesApi;
+  fully_diluted_valuation: CurrenciesApi;
+  circulating_supply: number;
+  total_supply: number;
+  max_supply: number;
+  current_price: CurrenciesApi;
+  price_change_24h: CurrenciesApi;
+  price_change_percentage_24h_in_currency: CurrenciesApi;
+};
+
+export type CoinMarketDataModels = {
```

Comment:
почему `Models`? это же тип одной сущности, а не массива
```suggestion
export type CoinMarketDataModel = {
```